doc/src/parsing_idioms.md

```diff
@@ -108,6 +123,69 @@ fn map_err(r: Result<Lexeme<StorageT>, Lexeme<StorageT>>)
 ```
 
 
+### Using `Err(Box<dyn Error>)`
+
+The idea here is that if we both stop evaluating normal action code, and
```

Comment:
remove the `if`

Reply:
Fixed in https://github.com/softdevteam/grmtools/pull/198/commits/211b301994666bcb5e2b7751efb7aa10b820afde.